tests/debugging/exception/test_replay.py

```diff
@@ -3,36 +3,56 @@
 import pytest
 
 import ddtrace
-import ddtrace.debugging._exception.auto_instrument as auto_instrument
+from ddtrace.debugging._exception import replay
 from ddtrace.internal.packages import _third_party_packages
 from ddtrace.internal.rate_limiter import BudgetRateLimiterWithJitter as RateLimiter
-from tests.debugging.mocking import exception_debugging
+from ddtrace.settings.exception_replay import ExceptionReplayConfig
+from tests.debugging.mocking import exception_replay
 from tests.utils import TracerTestCase
 
 
+def test_exception_replay_config_enabled(monkeypatch):
+    monkeypatch.setenv("DD_EXCEPTION_REPLAY_ENABLED", "1")
+
+    er_config = ExceptionReplayConfig()
+    assert er_config.enabled
+
+
+def test_exception_replay_config_enabled_deprecated(monkeypatch):
+    monkeypatch.setenv("DD_EXCEPTION_DEBUGGING_ENABLED", "1")
+
+    er_config = ExceptionReplayConfig()
+    assert er_config.enabled
```

Comment:
I'm assuming this is some envier magic that allows users to still use `DD_EXCEPTION_DEBUGGING_ENABLED` until it's unsupported in 3.x?

Reply:
Yes, if the actual variable is not specified, envier looks for the `deprecations`. If any are set, they are used instead and a deprecation warning is emitted